tests/foreach/foreach.py

```diff
@@ -0,0 +1,12 @@
+#! /usr/bin/env python
```

Comment:
As this is essentially a Python 3 script, use `python3` in the shebang. Besides, do we actually need this script? It does exactly the same thing as `foreach.sh` (which is actually employed by the cram test, as this one is not used by anyone).

Reply:
Removed in https://github.com/comelz/quark/pull/27/commits/8d9c9d80aaceda136592297c354071954f8c8221

The script was created to test the `quark foreach` command directly passing a python script to it.
As it isn't actually usefull I've directly removed it.